### ISSUE TITLE
Figure out something for extras

### ISSUE THREAD

**Issue by crimsonknave** [crimsonknave/obsidian_portal_fate#32]
Possibly something like stunts have with multiple rows that can be turned on and off. This will probably require some thought.

**Comment by crimsonknave** [comment]
Given that there are many existing character sheets that presumably are already using the simple text box for extras, I think the best solution would be to add additional fields like stunts that are all disabled by default. Then, the existing extras field would also have a `display` toggle.

**Comment by randypinion** [comment]
Agreed that multiple hideable (or dynamic add-able) textboxes would be an excellent addition for this game system. While the Fate Core rules don't make heavy use of Extras, the various Toolkits and especially derivative systems like Mindjammer and Bulldogs are very reliant on them, and characters could easily accrue 3-4 complex Extras over the course of a campaign.
